src/main/java/fr/techad/edc/popover/utils/OpenUrlAction.java

```diff
@@ -39,11 +40,23 @@ public OpenUrlAction(Browser browser, HelpConfiguration helpConfiguration) {
      */
     public void openUrl(String url) throws IOException, URISyntaxException {
         LOGGER.debug("Open the url: {}", url);
-        if (helpConfiguration.isInternalBrowser()) {
+
+        if(helpConfiguration.getHelpViewer() == HelpViewer.DESKTOP_VIEWER){
+            Runtime runtime = Runtime.getRuntime();
+            try
+            {
+                // Here write the good path to executable Desktop help viewer app
+                runtime.exec("C:\\Users\\Desktop\\EDC help viewer\\EDC help viewer.exe");
```

Comment:
Are you sure all people will install the viewer in this folder?
This library is deployed on central maven server and can be used as is. If the path can or must be modified, it should be defined from the configuration.